llama_stack/templates/ollama/run.yaml

```diff
@@ -16,19 +16,9 @@ providers:
     provider_type: remote::ollama
     config:
       url: ${env.OLLAMA_URL:http://localhost:11434}
-  - provider_id: sentence-transformers
```

Comment:
Hey @ashwinb!
Just out of curiosity, could you pls clarify the reason for removing the embedding model and the `sentence-transformers` provider from the `ollama` template?